reference/test/solver/multigrid_kernels.cpp

```diff
@@ -439,36 +448,14 @@ class Multigrid : public ::testing::Test {
                 .on(this->exec));
     }
 
-    std::unique_ptr<typename Solver::Factory> get_kcycle_factory(
```

Comment:
K cycle was dropped this is why all this is gone, right?

Reply:
Yes